bgpd/bgp_route.c

```diff
@@ -7260,15 +7258,16 @@ void route_vty_out(struct vty *vty, struct prefix *p,
 		if (json_paths) {
 			json_nexthop_global = json_object_new_object();
 
-			json_object_string_add(
-				json_nexthop_global, "afi",
-				nexthop_fqdn ? "fqdn"
-					     : (af == AF_INET) ? "ip" : "ipv6");
-			json_object_string_add(
-				json_nexthop_global,
-				nexthop_fqdn ? "fqdn"
-					     : (af == AF_INET) ? "ip" : "ipv6",
-				nexthop_fqdn ? nexthop_fqdn : nexthop);
+			json_object_string_add(json_nexthop_global, "ip",
+					       nexthop);
+
+			if (nexthop_fqdn)
+				json_object_string_add(json_nexthop_global,
+						       "fqdn", nexthop_fqdn);
+
+			json_object_string_add(json_nexthop_global, "afi",
```

Comment:
value of af is changed from ip to ipv4 for IPV4 address family. "ipv4" makes more sense than "ip", but if anyone has automations scripts based on this, then the scripts will fail.

Reply:
@srimohans basically yes, I changed `ip` --> `ipv4`. I could change this back to `ip`. How would you suggest doing here?

Reply:
I prefer `ipv4` here as well... I don't know what to do about other folks' scripts, though...

Reply:
I prefer `ipv4` here, as well... I don't know what to do about breaking scripts, but `ipv4` would be better in the long term.

Reply:
+1